### ISSUE TITLE
关于在paddlex 3.0 中 CocoDetection API的问题

### ISSUE THREAD

**Issue by fatsheep2020** [PaddlePaddle/PaddleX#2656]
## Checklist:

- [ ] 查找[历史相关issue](https://github.com/PaddlePaddle/PaddleX/issues)寻求解答
- [ ] 翻阅[FAQ](https://paddlepaddle.github.io/PaddleX/main/FAQ.html)
- [ ] 翻阅[PaddleX 文档](https://paddlepaddle.github.io/PaddleX/main/index.html)
- [ ] 确认bug是否在新版本里还未修复

## 描述问题
在paddlex 2.1中的API  paddlex.datasets.CocoDetection() 在3.0中没有了 那这个函数如何在3.0中实现呢？要自己手写吗？

urgent, thanks!

## 复现
example:
# 定义训练和验证所用的数据集
# API说明：https://github.com/PaddlePaddle/PaddleX/blob/develop/paddlex/cv/datasets/coco.py#L26
train_dataset = pdx.datasets.CocoDetection(
    data_dir='./train/images',
    ann_file='./instances_train.json',
    transforms=train_transforms,
    shuffle=True)
eval_dataset = pdx.datasets.CocoDetection(
    data_dir='./test/images',
    ann_file='./instances_val.json',
    transforms=eval_transforms)

1. 您是否已经正常运行我们提供的[教程](https://paddlepaddle.github.io/PaddleX/main/index.html)？

2. 您是否在教程的基础上修改代码内容？还请您提供运行的代码

3. 您使用的数据集是？

4. 请提供您出现的报错信息及相关log

## 环境

1. 请提供您使用的PaddlePaddle和PaddleX的版本号
3.0
2. 请提供您使用的操作系统信息，如Linux/Windows/MacOS
ubuntu
3. 请问您使用的Python版本是？

5. 请问您使用的CUDA/cuDNN的版本号是？

**Comment by TingquanGao** [comment]
3.0升级改动较大，和2.0的使用逻辑不同。是有什么具体的需求呢？可以先参考下新的文档：https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta2/docs/module_usage/tutorials/cv_modules/object_detection.md#41-%E6%95%B0%E6%8D%AE%E5%87%86%E5%A4%87

**Comment by fatsheep2020** [comment]
i have descripted my question above, can we still use this API in 3.0? paddlex.datasets.CocoDetection() 
@TingquanGao  thanks!

**Comment by TingquanGao** [comment]
这个接口在3.0不支持了

**Comment by wangwenqi567** [comment]
小白想请教一下，离线下载模型之后，create_model还是会从网址上下载模型，应该怎么解决

**Comment by TingquanGao** [comment]
> 小白想请教一下，离线下载模型之后，create_model还是会从网址上下载模型，应该怎么解决

这个问题到新的issue跟进吧 #2715

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
